src/lib/components/filegrid.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+    import { RecursiveTreeView } from "@skeletonlabs/skeleton";
+    
+    /**
+     * @type {import("@skeletonlabs/skeleton").TreeViewNode[]}
+     */
+    export let nodes;
+    /**
+     * @type {string[]}
+    */
+    let checkedNodes;
+    /**
+     * @type {string[]}
+    */
+    let indeterminateNodes;
+
+    // To show checkedNodes & indeterminateNodes
+    // function handleClick() {
+    // console.log('Checked Nodes:', checkedNodes);
+    // console.log('Indeterminate Nodes:', indeterminateNodes);
+    //   }
+                            
+</script>
+
+<div class = 'w-1/2'>
+<RecursiveTreeView selection multiple relational {nodes} bind:checkedNodes={checkedNodes} bind:indeterminateNodes={indeterminateNodes} />
+</div>
+<!-- To show checkedNodes & indeterminateNodes -->
+<!-- <RecursiveTreeView selection multiple relational {nodes} bind:checkedNodes={checkedNodes} bind:indeterminateNodes={indeterminateNodes} on:click={handleClick} /> -->
```

Comment:
I think you can remove the comments. it should be obvious enough, that checkedNodes and indeterminateNodes are accessible.

Reply:
Just had it there for testing